enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -751,6 +779,10 @@ func (p *PebbleCache) maxDatabaseVersion() filestore.PebbleKeyVersion {
 	return p.maxDBVersion
 }
 
+func (p *PebbleCache) setActiveDatabaseVersion(version filestore.PebbleKeyVersion) {
```

Comment:
does this need to be its own func? seems like you could initialize it directly in NewPebbleCache

Reply:
Nope, removed.